### PR TITLE
Sc-18436: Member Status Check & minor improvement

### PR DESCRIPTION
**Pull request by masskoder**
### Scope of changes

  This adds member status logic and few components improvements

### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: web/gds-user-ui/src/modules/dashboard/member/MemberTable.tsx [hunk 1]
```diff
@@ -1,81 +1,44 @@
...
-            </Th>
-          </Tr>
-        </Thead>
-        <Tbody>
-          <MemberTableRows rows={data} />
-        </Tbody>
-      </Table>
-    </FormLayout>
+    <Table variant="simple">
+      <TableCaption placement="top" textAlign="end" p={0} m={0} mb={3} fontSize={20}></TableCaption>
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/modules/dashboard/member/MemberTable.tsx:15]: Will something be added to the `TableCaption`?

File: web/gds-user-ui/src/modules/dashboard/member/components/MemberTable.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import MemberTableList from './MemberTableList';
+
+// interface MemberTableProps {
+//   data: any;
+// }
+
+const MemberTable: React.FC = () => {
+  return (
+    <Table variant="simple">
+      <TableCaption placement="top" textAlign="end" p={0} m={0} mb={3} fontSize={20}></TableCaption>
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/modules/dashboard/member/components/MemberTable.tsx:15]: Also, will this `TableCaption` have something added to it?

File: web/gds-user-ui/src/modules/dashboard/member/components/MemberTableList.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import MemberTableRows from './MemberTableRows';
+import { useFetchMembers } from '../hooks/useFetchMembers';
+
+const MemberTableList = () => {
+  const { error, members, isFetchingMembers } = useFetchMembers();
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/modules/dashboard/member/components/MemberTableList.tsx:6]: This is a really good change!

File: web/gds-user-ui/src/modules/dashboard/member/components/MemberTableRow.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import { Td, Tr, Button, HStack, chakra, Tag } from '@chakra-ui/react';
+import { BsEye } from 'react-icons/bs';
+import { formatIsoDate } from 'utils/formate-date';
+import { getVapsNetwork } from '../utils';
+import { VaspType } from '../memberType';
+
+const MemberTableRow: React.FC<{ row: VaspType }> = ({ row }) => {
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/modules/dashboard/member/components/MemberTableRow.tsx:7]: Thanks for making this change and for adding the `MemberTableRows` component. Separating the components has made it easier to follow what's happening.

**Review by daniellemaxwell — APPROVED** [review]
The table is getting much closer to it's final state. Thanks for all the effort that went into this one!

File: web/gds-user-ui/src/modules/dashboard/member/MemberTable.tsx (hunk 1 above)
Comment (masskoder) [reply web/gds-user-ui/src/modules/dashboard/member/MemberTable.tsx:15]: I think we should probably remove it. it will be done in my current PR. thanks

**Pull request merged**